packages/dataviews/src/types.ts

```diff
@@ -73,7 +73,7 @@ export type Field< Item > = {
 	/**
 	 * Callback used to render the field. Defaults to `field.getValue`.
 	 */
-	render?: ( args: { item: Item } ) => ReactNode;
+	render?: ComponentType< { item: Item } >;
```

Comment:
updating the type actually forces us to use the property properly and prevents us from doing the same error in the future.